src/pages/apps/_components/AppHeader.tsx

```diff
@@ -11,13 +11,17 @@ interface Props extends Pick<RootContextProps, "api"> {
 
 const AppHeader: React.FC<Props> = ({ app, envs, api }): React.ReactElement => {
   const provider = app.repo.split("/").shift();
-
+  const repoUrl = (app: App): string =>
+    `https://${app.provider}.com/${app.name}`;
+    console.log("--->", repoUrl(app));
   return (
     <header className="flex w-full mt-6">
       <div className="flex items-center pt-1">
         <span className={`fab fa-${provider} text-white mr-6 text-5xl`} />
         <div className="flex flex-col">
-          <div className="text-white mb-1">{app.name}</div>
+          <div className="text-white mb-1">
+            <a href={repoUrl(app)}> {app.repo} </a>
```

Comment:
I tried to use <Link> component but that is used for routing, that doesn't generate right html for linking repository on github

Reply:
Hmm weird, it should: 

```
if (to.indexOf("http") === 0 || to.indexOf("//") === 0) {
      rest.rel = "noreferrer noopener";
      rest.target = "_blank";
      isExternal = true;
    } else if (to.indexOf("mailto:") === 0) {
      isExternal = true;
    }

    if (isExternal) {
      return (
        <a href={to} {...rest} ref={ref} className={classes}>
          {children}
        </a>
      );
    }
```